index.js

```diff
@@ -124,32 +106,11 @@ function update (url) {
 
   return Promise.all([
     manifestPromise,
-    torrentPromise
+    torrentPromise,
+    priority.clear()
```

Comment:
Yes.

The install event that calls update is only fired when the service worker file is changed. This check can happen frequently but it never fires the event unless the file is different